packages/app-store/ee/routing-forms/components/react-awesome-query-builder/widgets.tsx

```diff
@@ -104,7 +104,6 @@ const MultiSelectWidget = ({
   return (
     <Select
       className="dark:border-darkgray-300 mb-2 block w-full min-w-0 flex-1 rounded-none rounded-r-sm border-gray-300 dark:bg-transparent dark:text-white dark:selection:bg-green-500 disabled:dark:text-gray-500 sm:text-sm"
-      menuPosition="fixed"
```

Comment:
Why are we removing this?

Reply:
The options from the select did show up at a random place in the new Dialog (not right under it). I tested the live form and it didn't change anything there. @hariombalhara Is this needed for anything that I am not aware of?